### PR TITLE
Add 'grant_registry_access' variable to add 'storage.objectViewer'-role to SA

### LINKED ISSUE #229: Grant storage viewer to created service accounts
I am creating a testing gke cluster using this module, everything works, but after the cluster is being created, docker fails to pull images from gcr private repository, because the service account created by this modules doesn't has the roles/storage.objectViewer

In the sa.tf I don't see nothing for the needed role:

```
resource "google_service_account" "cluster_service_account" {
  count        = var.service_account == "create" ? 1 : 0
  project      = var.project_id
  account_id   = "tf-gke-${substr(var.name, 0, min(15, length(var.name)))}-${random_string.cluster_service_account_suffix.result}"
  display_name = "Terraform-managed service account for cluster ${var.name}"
}

resource "google_project_iam_member" "cluster_service_account-log_writer" {
  count   = var.service_account == "create" ? 1 : 0
  project = google_service_account.cluster_service_account[0].project
  role    = "roles/logging.logWriter"
  member  = "serviceAccount:${google_service_account.cluster_service_account[0].email}"
}

resource "google_project_iam_member" "cluster_service_account-metric_writer" {
  count   = var.service_account == "create" ? 1 : 0
  project = google_project_iam_member.cluster_service_account-log_writer[0].project
  role    = "roles/monitoring.metricWriter"
  member  = "serviceAccount:${google_service_account.cluster_service_account[0].email}"
}

resource "google_project_iam_member" "cluster_service_account-monitoring_viewer" {
  count   = var.service_account == "create" ? 1 : 0
  project = google_project_iam_member.cluster_service_account-metric_writer[0].project
  role    = "roles/monitoring.viewer"
  member  = "serviceAccount:${google_service_account.cluster_service_account[0].email}"
}

```

### PR DESCRIPTION
**Pull request by paulpalamarchuk**
* Added `grant_registry_access` variable to grant `roles/storage.objectViewer` to created SA (Fixes #229)

### CODE REVIEW

**Comment by kopachevsky** [comment]
@morgante @aaron-lane can you cleanup CI server please

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Extending the adopted spec, each change should have a link to its corresponding
 ## [Unreleased]
 ### Added
 
+* Added `sa_grant_gcs_access` variable to grant `roles/storage.objectViewer` to created SA [#236]
+### Added
```
Comment (kopachevsky) [inline CHANGELOG.md]: remove **### Added** duplication

**Comment by morgante** [comment]
@paulpalamarchuk Let's call the variable `grant_registry_access`.

**Comment by paulpalamarchuk** [comment]
> @paulpalamarchuk Let's call the variable `grant_registry_access`.

@morgante Variable name updated.

**Pull request merged**